src/css/_header.scss

```diff
@@ -71,6 +71,10 @@ header {
   height: 40px;
 }
 
+.choices__input {
+  font-size: 16px;
+}
```

Comment:
Can you please add a brief comment linking to the issue https://github.com/ParkingReformNetwork/parking-lot-map/issues/177?

Reply:
Wouldn't that be accessible via the Git/GitHub history? I was under the impression most folks are using GitLens these days.

Reply:
Yes, it is, but it's not safe to assume that everyone is using GitLens, and this repo has many newer developers. The comment is meant to make clear that this isn't simply a stylistic choice, but to fix a non-obvious bug.